smartpower3/screens/settingscreen.cpp

```diff
@@ -412,70 +437,60 @@ void SettingScreen::changeSerialBaud(uint8_t baud_level)
 {
 	drawSerialBaud(serial_value[baud_level]);
 	serial_baud_edit = serial_value[baud_level];
-	changeLogInterval(log_interval);
-}
-
-void SettingScreen::selectBLLevel(uint16_t color)
-{
-	for (int i = 1; i < SELECTION_BORDER_WIDTH+1; i++) {
-		tft->drawRect(x + X_BL_LEVEL-i, y-1-i, 135+i*2, 28+i*2, color);
-	}
+	changeLogInterval(log_interval, true);
 }
 
-void SettingScreen::selectSerialLogging(uint16_t color)
```

Comment:
This was the method that drew the frame around both serial baud and logging interval.